docs/schema/action-element-in-actions-bdcmetadata-schema.md

```diff
@@ -98,60 +98,22 @@ The following sections describe attributes, child elements, and parent elements.
 </tbody>
 </table>
 
-<br/>
+</br>
 
 ### Child elements
+  
+| Element | Description |
```

Comment:
This just caught my attention, but I am not sure if it makes a difference, but does it matter if there are extra spaces here?

Reply:
It makes no difference, but it can be fixed if desired.